Makefile

```diff
@@ -191,6 +214,9 @@ install-protoc install-protobuf:
 install-ttrpc-plugin:
 	$(Q)$(GO_INSTALL) -mod=mod github.com/containerd/ttrpc/cmd/protoc-gen-go-ttrpc@74421d10189e8c118870d294c9f7f62db2d33ec1
 
+install-wasm-plugin:
+	$(Q)$(GO_INSTALL) -mod=mod github.com/knqyf263/go-plugin/cmd/protoc-gen-go-plugin@d8d42059d8f1b52968cff7226b7094e5c6a0c342
```

Comment:
The hashicorp variant has one huge drawback, because it's launching a plugin as a subprocess which is not secure. In addition, plugin authors need to distribute multi-arch binaries. That's why I decided to go with (the only) alternative right now.